tests/unit/spec/AdalSpec.js

```diff
@@ -586,6 +586,26 @@ describe('Adal', function () {
         expect(storageFake.getItem(adal.CONSTANTS.STORAGE.USERNAME)).toBeUndefined();
     });
 
+    it ('test decode with no padding', function () {
+        expect(adal._decode('ZGVjb2RlIHRlc3Rz')).toBe('decode tests');
```

Comment:
@weijjia But the padding is directly related to the length of the string that is encoded.  I think it would be best to match those things up.